Backend using .NET Core 2.2 dependencies

LGTM is failing C# builds due to:

```
[2020-06-12 17:04:58] [build] (CoreCompile target) -> 
[2020-06-12 17:04:58] [build]   CSC : error CS1705: Assembly 'BackendFramework' with identity 'BackendFramework, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' uses 'Microsoft.AspNetCore.Mvc.ViewFeatures, Version=3.1.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' which has a higher version than referenced assembly 'Microsoft.AspNetCore.Mvc.ViewFeatures' with identity 'Microsoft.AspNetCore.Mvc.ViewFeatures, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' [/opt/src/Backend.Tests/Backend.Tests.csproj]
[2020-06-12 17:04:58] [build]   CSC : error CS1705: Assembly 'BackendFramework' with identity 'BackendFramework, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' uses 'Microsoft.AspNetCore.Http.Abstractions, Version=3.1.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' which has a higher version than referenced assembly 'Microsoft.AspNetCore.Http.Abstractions' with identity 'Microsoft.AspNetCore.Http.Abstractions, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' [/opt/src/Backend.Tests/Backend.Tests.csproj]
[2020-06-12 17:04:58] [build]     16 Warning(s)
[2020-06-12 17:04:58] [build]     2 Error(s)
```

This appears to be because of:

https://github.com/sillsdev/TheCombine/blob/ae022a6b4b87863042f3720fdec0d6cc0aaee73e/Backend.Tests/Backend.Tests.csproj#L14-L15

--- a/Backend.Tests/Backend.Tests.csproj
+++ b/Backend.Tests/Backend.Tests.csproj
@@ -11,8 +11,6 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Abstractions" Version="2.2.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.ViewFeatures" Version="2.2.0" />
     <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="5.5.0" />
     <PackageReference Include="MongoDB.Driver" Version="2.9.0-beta2" />
     <PackageReference Include="nunit" Version="3.12.0" />

